src/UI/Component/Popover/Popover.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace ILIAS\UI\Component\Popover;
+
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Signal;
+use ILIAS\UI\Component\Triggerable;
+
+/**
+ * Describes the Popover component
+ */
+interface Popover extends Component, Triggerable {
+
+	/**
+	 * Get the title of the popover
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Get the text (content) of the popover
+	 *
+	 * @return string
+	 */
+	public function getText();
+
+	/**
+	 * Get the position of the popover: top, bottom, left, right or auto
+	 *
+	 * @return string
+	 */
+	public function getPosition();
```

Comment:
Unless someone has a use case where its definitely necessary that the Dev decides, I would just drop this position property and always set it to auto. I currently can't see that use case.